src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/MapAction_ExplicitSource_SimpleReturn_Snapshot.generated.txt

```diff
@@ -534,6 +534,12 @@ namespace Microsoft.AspNetCore.Http.Generated
                     return (false, default);
                 }
             }
+
+            if (!allowEmpty)
```

Comment:
First time looking at this code/area so maybe I missed something, but this is logically an `else if` right? Doesn't seem to be anyway to fall out of the prior `if` block once you're in it.

If so, consider making this an `else if` for clarity (here and in other places).

Reply:
Yep, this is the designed to handle the case where you've specified that a JSON body is required in this request (`allowEmpty = false`) but the `IHttpRequestBodyDetectionFeature` didn't detect a body in the request because there was no content length set.